tests/test_widgets.py

```diff
@@ -0,0 +1,17 @@
+from prompt_toolkit.formatted_text import fragment_list_to_text
+from prompt_toolkit.layout import to_window
+from prompt_toolkit.widgets import Button
+
+
+class TestButton:
```

Comment:
Thanks you for this PR!

One proposal on this for consistency. Would it be possible to not use a class, and make all three methods top-level.
I don't think any of the other unit tests use classes, but rather top-level methods and fixtures.
In this case, the class is only used as a namespace, which is functionality already provided by the module.

Thanks! (I'll merge it if you can change this.)

Reply:
Changed!